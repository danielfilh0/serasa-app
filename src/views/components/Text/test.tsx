import { render, screen } from '@testing-library/react'

import { Text } from '.'

describe('<Text />', () => {
  it('should render the heading', () => {
    const { container } = render(<Text>Body</Text>)

    expect(screen.getByText('Body')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
