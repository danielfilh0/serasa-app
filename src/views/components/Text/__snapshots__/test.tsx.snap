// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> should render the heading 1`] = `
<p
  class="font-normal text-dark-high text-base"
>
  Body
</p>
`;
